[2024-02-24 18:13:47,077] [INFO] [server_logger.py:29] Logger initialized
[2024-02-24 18:13:47,114] [ERROR] [login.py:25] Error during connection: 1045 (28000): Access denied for user 'root'@'localhost' (using password: NO)
[2024-02-24 18:13:47,138] [ERROR] [register.py:20] Error during connection: 1045 (28000): Access denied for user 'root'@'localhost' (using password: NO)
[2024-02-24 18:13:47,161] [ERROR] [survey.py:16] Error during connection: 1045 (28000): Access denied for user 'root'@'localhost' (using password: NO)
